include/state_machine.h

```diff
@@ -30,10 +30,17 @@ class StateMachine
   using state_iterator = std::list<State>::iterator;
 
   void constructObstacleMenu(State* from);
-  void constructWalkMenu(state_iterator lastState);
-  void constructStepMenu(state_iterator lastState);
+  void constructWalkMenu(state_iterator lastState, State* obstacles);
+  void constructStepMenu(state_iterator lastState, State* obstacles);
   void constructSofaMenu(State* from, State* next_obstacle);
+  // void constructWalkSizeMenu(State* from, State* prev_gait);
+  // void constructStepSizeMenu(State* from, State* prev_gait);
+  // void constructSideStepMenu(State* from);
```

Comment:
Please remove the function definitions if they are no longer used

Reply:
Oh right, removed them from the cpp file, but forgot about the header